styles/Canonical/013-Spell-out-numbers-below-10.yml

```diff
@@ -0,0 +1,21 @@
+extends: substitution
+message: "Consider using '%s' instead of '%s'"
+link: https://docs.ubuntu.com/styleguide/en#numbers
+scope:
```

Comment:
Out of curiosity, is there a particular reason for us not to include caption text? (`table.caption` and `figure.caption`)

Reply:
It is mostly a side effect of Rule 14 where defining the scope was tricky and we ended up choosing the currently defined scope (did not want to create too many false positives for rule 14). I don't see an issue in adding them or leaving them out for this rule. What do others think?